test/apps/activitycallee/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
```

Comment:
Regarding to two these test apps: ```activity_caller``` and ```activity_callee``` - we use the same approach in in gaia/sms to avoid dependency on real applications. Especially in the case when we need to call an activity - for real application it would be a number of steps that highly depends on the app. Please, tell me what you think about it? Is there better way to do it?